.github/workflows/nightly_trigger.yaml

```diff
@@ -76,3 +76,19 @@ jobs:
           gh workflow run raspi-2_20.lts.1+ --ref 20.lts.1+ -f nightly=true
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  trigger_19:
+    permissions: write-all
```

Comment:
This is a gonna give the token a permission to write for all of the available GitHub scopes, do we want this behavior?

Reply:
Do you happen to know which permission specifically needed to be able to trigger workflows, I haven't been able to pin point specific permission?